samples/gtk/README.md

```diff
@@ -3,18 +3,33 @@
  This example shows how one may use _Kotlin/Native_ to build GUI
  applications with the GTK toolkit.
 
-To build use `../gradlew build` or `./build.sh`.
+To build use `../gradlew build` or `./build.sh [-I=/include/path]`.
 
-Do not forget to install GTK3.
+Do not forget to install GTK3. See bellow.
 
 On Mac use `port install gtk3`, on Debian flavours of Linux - `apt-get install libgtk-3-dev`.
 To run on Mac also install XQuartz X server (https://www.xquartz.org/), and then
 
     ../gradlew run
-    
-Alternatively you can run artifact directly 
+
+Alternatively you can run artifact directly
 
     ./build/konan/bin/Gtk3Demo/Gtk3Demo.kexe
 
 Dialog box with the button will be shown, and application will print message
 and terminate on button click.
+
+
+#### GTK3 Install
```

Comment:
This has to be combined with the text above: "On Mac use `port install gtk3`, on Debian flavours of Linux - `apt-get install libgtk-3-dev`." and use uppercased proper nouns.

proper noun |  
-- | --




proper noun |  
-- | --